cleancat/sqla.py

```diff
@@ -38,3 +41,28 @@ def fetch_existing(self, pk):
 
     def get_orig_data_from_existing(self, obj):
         return object_as_dict(obj)
+
+
+class SQLAReference(Field):
+    """
+    Represents a reference to an SQLAlchemy object. Expects an ID string as
+    input and returned a cleaned model instance (verifying that it exists
+    first).
+    """
+
+    base_type = str_type
```

Comment:
A common use case is numeric (auto-increment) IDs. I haven't tested this, but would you potentially also want to support `int` here?